specs/ChainSpec/Consensus.md

```diff
@@ -99,6 +101,8 @@ def process_timer(self):
 
 Where `send_approval` creates an approval with `prev_hash` and `prev_height` equal to `self.head_hash` and `self.head_height` correspondingly, `target_height` to the passed argument, and sends to the block producer at `target_height`.
```

Comment:
The fields `prev_height` and `prev_hash` are gone now, so every mention of them should be reworded.